<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>572</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>CortexView 2</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/new/icon/img/icon-cortex-view.png</normaloff>:/new/icon/img/icon-cortex-view.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QWidget" name="controlPanel" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>512</y>
      <width>512</width>
      <height>60</height>
     </rect>
    </property>
    <widget class="QPushButton" name="recordButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Start/End recording session</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="icons.qrc">
       <normaloff>:/new/icon/img/icon-record-alt.png</normaloff>
       <normalon>:/new/icon/img/icon-record-inactive.png</normalon>
       <activeon>:/new/icon/img/icon-stop-inactive.png</activeon>
       <selectedon>:/new/icon/img/icon-record-alt.png</selectedon>:/new/icon/img/icon-record-alt.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="resetROIButton">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Reset ROI to full screen</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normalon>:/new/icon/img/icon-reset-roi.png</normalon>
      </iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>14</y>
       <width>82</width>
       <height>12</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Recording Progress</string>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QProgressBar" name="temperatureBar">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>34</y>
       <width>82</width>
       <height>12</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Camera Temperature</string>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>50</number>
     </property>
     <property name="value">
      <number>-20</number>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="format">
      <string>%vC</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cameraButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>10</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Take still image</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="icons.qrc">
       <normaloff>:/new/icon/img/icon-take-still.png</normaloff>:/new/icon/img/icon-take-still.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QSpinBox" name="cycleNoSpin">
     <property name="geometry">
      <rect>
       <x>419</x>
       <y>34</y>
       <width>45</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::PlusMinus</enum>
     </property>
     <property name="accelerated">
      <bool>true</bool>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>80</number>
     </property>
     <property name="singleStep">
      <number>5</number>
     </property>
     <property name="value">
      <number>20</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="filePathEdit">
     <property name="geometry">
      <rect>
       <x>252</x>
       <y>8</y>
       <width>177</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QSpinBox" name="periodSpinInF">
     <property name="geometry">
      <rect>
       <x>252</x>
       <y>34</y>
       <width>50</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::PlusMinus</enum>
     </property>
     <property name="accelerated">
      <bool>true</bool>
     </property>
     <property name="minimum">
      <number>6</number>
     </property>
     <property name="maximum">
      <number>2000</number>
     </property>
     <property name="singleStep">
      <number>6</number>
     </property>
     <property name="value">
      <number>900</number>
     </property>
    </widget>
    <widget class="QPushButton" name="browseFileButton">
     <property name="geometry">
      <rect>
       <x>427</x>
       <y>8</y>
       <width>75</width>
       <height>18</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>browse...</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="periodSpinInS">
     <property name="geometry">
      <rect>
       <x>348</x>
       <y>34</y>
       <width>55</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::PlusMinus</enum>
     </property>
     <property name="accelerated">
      <bool>true</bool>
     </property>
     <property name="value">
      <double>12.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>304</x>
       <y>34</y>
       <width>40</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>frames</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>405</x>
       <y>34</y>
       <width>10</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>s</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>466</x>
       <y>34</y>
       <width>36</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>cycles</string>
     </property>
    </widget>
   </widget>
   <widget class="CameraView" name="cameraView" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>512</width>
      <height>512</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CameraView</class>
   <extends>QWidget</extends>
   <header>cameraview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
